### PR TITLE
Review provenance to have a better translation in French

### PR DESCRIPTION
**Pull request by Gustry**
### What does it fix ?
* Ticket #
* Funded by : WorldBank
* Description :

### CODE REVIEW

File: safe/definitions/hazard_exposure_specifications.py [hunk 1]
```diff
@@ -55,7 +63,15 @@
                 'model.'),
             # end notes provided by Hadi Ghasemi
         ]
-    }
+    },
+    {
+        'hazard': hazard_flood,
+        'exposure': exposure_road,
+        'analysis_question': {
+            'fr': u"Dans l'événement d'une inondation, quelle longueur de "
```
Comment (lucernae) [inline safe/definitions/hazard_exposure_specifications.py]: I'm just asking the possibility here.
Does it have to be hardcoded?
Or can we use tr('Some unique statement about this case'),
then translate it (put the translated string) on transifex?

File: safe/impact_function/impact_function.py [hunk 2]
```diff
@@ -856,12 +855,12 @@ def prepare(self):
                 return status, message
 
             # Set the name
-            self._name = tr('%s %s On %s %s' % (
-                self.hazard.keywords.get('hazard').title(),
-                self.hazard.keywords.get('layer_geometry').title(),
-                self.exposure.keywords.get('exposure').title(),
-                self.exposure.keywords.get('layer_geometry').title(),
-            ))
+            self._name = tr('%s %s On %s %s') % (
```
Comment (lucernae) [inline safe/impact_function/impact_function.py]: can we use {name} format on this (py3 style with named variable)?

File: safe/impact_function/impact_function.py [hunk 3]
```diff
@@ -1997,6 +1996,8 @@ def provenance(self):
         If the impact function is not ready (has not called prepare method),
         it will return empty dict to avoid miss information.
 
+        The impact function will call generate_provenance at the end of the IF.
+
         List of keys (for quick lookup):
 
         [
```
Comment (lucernae) [inline safe/impact_function/impact_function.py]: @Gustry and @ismailsunni It's hard to keep track what can be expected to be inside the provenance for now.
So, if possible, if there's any changes in the future, please change this list accordingly (for example, we might want to delete report_question?).
Best solution would be to declare another container class for provenance (for self._provenance object).
That way, pycharm can detect its properties without having to run it.

Comment (Gustry) [inline safe/impact_function/impact_function.py]: Yes, agree!

File: safe/definitions/hazard_exposure_specifications.py (hunk 1 above)
Comment (Gustry) [inline safe/definitions/hazard_exposure_specifications.py]: I didn't want you to translate this kind of strings on transifex. If you want, we can put the 'tr' if you think we need it. But as you told me these generated strings works well for you, I though it's fine.

**Pull request merged**